controllers/posts.js

```diff
@@ -15,6 +15,15 @@ const PostsController = {
   },
   Create: (req, res) => {
     const post = new Post(req.body);
+    // const datepost = (req._startTime); // gets time/date from message body
+    const todaysdate = Date().slice(0, -31); // gets time/date from mongoose
+    // console.log(todaysdate);
```

Comment:
I think as Michael it would be good to do this counting from the beginning onwards in case the final words change
You can also use the substr method to do this - .substr(0, 11) for example